crates/ruma-identity-service-api/src/association/unbind_3pid/v2.rs

```diff
@@ -0,0 +1,67 @@
+//! [POST /_matrix/identity/v2/3pid/unbind](https://matrix.org/docs/spec/identity_service/r0.3.0#post-matrix-identity-v2-3pid-unbind)
+
+use ruma_api::ruma_api;
+use ruma_common::thirdparty::Medium;
+use ruma_identifiers::user_id::UserId;
+use serde::{Deserialize, Serialize};
+
+ruma_api! {
+    metadata: {
+        description: "Remove an association between a session and a Matrix user ID.",
+        method: POST,
+        name: "unbind_3pid",
+        path: "/_matrix/identity/v2/3pid/unbind",
+        rate_limited: false,
+        authentication: AccessToken,
```

Comment:
I assumed that an access token was necessary in both cases. I've asked about it (and about your other comment) to make sure in the #matrix-spec channel.

Reply:
The current answer in the issue I opened isn't very conclusive about the need of a token. However, it seems that the swagger for the endpoint states an access token is needed.
Looking at other endpoints in the IS spec, endpoints do not require a token in their `v1` variant, and do require one in the `v2` variant. So to be consistent with the rest of the `v2` endpoints currently implemented in Ruma, we may want to require an `AccessToken`.

Reply:
Sure, it's probably better to default to being more secure (i.e. `AccessToken` instead of `MaybeAccessToken`). I wish that we could get some more concrete answers on this one, but this will have to do for now.